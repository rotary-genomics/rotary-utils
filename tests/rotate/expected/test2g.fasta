>sequence1
CCTTTTTGGGGGAAAAACCC
>sequence2 # comment
AGGGTTTCCCAA
>sequence3
GGAATTCC
