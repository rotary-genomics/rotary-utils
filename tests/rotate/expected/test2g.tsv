sequence_id	sequence_length_bp	rotate_type	rotate_value
sequence1	20	position	8
sequence2	12	position	8
sequence3	8	position	8
